content/en/aws/docdb/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "DocumentDB (DocDB)"
+linkTitle: "DocumentDB (DocDB)"
+categories: ["LocalStack Pro"]
+description: >
+  DocumentDB (DocDB)
+---
+
+LocalStack supports a basic version of [Amazon DocumentDB](https://aws.amazon.com/documentdb/) for testing.
+
+When loading the DocumentDB service, LocalStack will create a MongoDB container to handle DocumentDB storage. By default, this instance will listen on a port in the 33000-34000 port range and will instantiate a master user named 'test' without a password. The `mongosh` client may be used to log in to and interact with the instance.
+
+{{< alert title="Note" >}}
+Because LocalStack utilizes a MongoDB container to provide DocumentDB storage, LocalStack may not have feature parity with Amazon DocumentDB. The database engine may support additional features that Amazon DocumentDB does not and vice versa.
+{{< /alert >}}
+
+## Example
+
+{{< command >}}
+$ awslocal docdb create-db-cluster --db-cluster-identifier 'test-docdb-cluster' --engine 'docdb'
```

Comment:
One final question. When LocalStack spins up the `mongo` container for the first time, I notice this stack trace in the debug logs.

```
localstack_main  | 2022-10-28T04:27:36.148 DEBUG --- [  Thread-121] l.u.c.docker_sdk_client    : Starting container 8cc4bbc94b2f690aed650dca9fc2f3bef0aabe511315266e5ea8247e39269b9c
localstack_main  | 2022-10-28T04:27:43.006  INFO --- [   asgi_gw_0] localstack.request.aws     : AWS rds.CreateDBInstance => 200
localstack_main  | 2022-10-28T04:28:18.081  WARN --- [functhread25] l.services.rds.db_utils    : Unable to startup DB instance: Port http://23.202.231.169:50761 (path: None) was not open Traceback (most recent call last):
localstack_main  |   File "/opt/code/localstack/.venv/lib/python3.10/site-packages/localstack_ext/services/rds/db_utils.py.enc", line 48, in B
localstack_main  |   File "/opt/code/localstack/.venv/lib/python3.10/site-packages/localstack_ext/services/docdb/docdb_api.py.enc", line 17, in start_db_instance
localstack_main  |   File "/opt/code/localstack/.venv/lib/python3.10/site-packages/localstack_ext/services/docdb/docdb_api.py.enc", line 24, in start_server
localstack_main  |   File "/opt/code/localstack/localstack/utils/net.py", line 93, in wait_for_port_open
localstack_main  |     return wait_for_port_status(
localstack_main  |   File "/opt/code/localstack/localstack/utils/net.py", line 133, in wait_for_port_status
localstack_main  |     return retry(check, sleep=sleep_time, retries=retries)
localstack_main  |   File "/opt/code/localstack/localstack/utils/sync.py", line 65, in retry
localstack_main  |     raise raise_error
localstack_main  |   File "/opt/code/localstack/localstack/utils/sync.py", line 61, in retry
localstack_main  |     return function(**kwargs)
localstack_main  |   File "/opt/code/localstack/localstack/utils/net.py", line 128, in check
localstack_main  |     raise Exception(
localstack_main  | Exception: Port http://23.202.231.169:50761 (path: None) was not open
localstack_main  |   
localstack_main  | 2022-10-28T04:29:58.835  INFO --- [   asgi_gw_1] localstack.request.aws     : AWS rds.DescribeDBClusters => 200
```

Despite this stacktrace, `mongo` appears to be working fine. I am able to log in to the DocumentDB instance, create collections, insert documents, etc. Is this something I should worry about? If so, how might I work around this issue so that I may include it in this documentation?

Reply:
That's weird - do you always see this stacktrace? 
I could not reproduce this issue locally, but from the stacktrace it seems that the port is not accessible after the pre-defined waiting-time. 
From what you describe, it seems like it only took longer than expected though, so we might need to increase the timeout.